docs/docs/overview/performance.md

```diff
@@ -22,13 +22,15 @@ We used the following configurations for the benchmarks.
 
 - Nodes: 3
 - Machines: `n2d-standard-2`
+- CVM: `true`
 - Kubernetes version: `1.23.6-gke.2200`
```

Comment:
That specific version with the GKE tag only applies to GKE, not Constellation.

Reply:
Yeah, this is mostly a result because throughout the edits, we kind of merged the setup specifications between cloud provided Kubernetes and Constellation.

Reply:
I think I might exclude the Kubernetes version here as it has probably little impact on the performance. But keep the rest.